src/screens/OrgSettings/OrgSettings.tsx

```diff
@@ -6,19 +6,32 @@ import UserUpdate from 'components/UserUpdate/UserUpdate';
 import OrgUpdate from 'components/OrgUpdate/OrgUpdate';
 import OrgDelete from 'components/OrgDelete/OrgDelete';
 import AdminNavbar from 'components/AdminNavbar/AdminNavbar';
+import MemberRequestCard from 'components/MemberRequestCard/MemberRequestCard';
+import { useQuery } from '@apollo/client';
+import { MEMBERSHIP_REQUEST } from 'GraphQl/Queries/Queries';
 
 function OrgSettings(): JSX.Element {
-  const [showUserUpdate, setShowUserUpdate] = React.useState(false);
-  const [showOrgUpdate, setShowOrgUpdate] = React.useState(false);
-  const [showOrgDelete, setShowOrgDelete] = React.useState(false);
+  const [t, setT] = React.useState(0);
```

Comment:
please use the full phrase/name for the variable/attribute you are controlling the state for.

`t` does not give any description as to what's being set or used